db_stress_tool/cf_consistency_stress.cc

```diff
@@ -1040,17 +1040,17 @@ class CfConsistencyStressTest : public StressTest {
     assert(thread);
     Status status;
 
-    DB* db_ptr = cmp_db_ ? cmp_db_ : db_;
```

Comment:
I guess the original others intended for `cmp_db_` to potentially be used for other purposes, but right now the only usages are for opening secondary databases. So I think we can improve the naming